src/Microsoft.DotNet.Archive/LZMA/ICoder.cs

```diff
@@ -0,0 +1,157 @@
+// ICoder.h
```

Comment:
Seems out of date.

More importantly - are we 100% sure that there is no copyright header that needs to be here?